apps/language_server/lib/language_server/experimental/source_file/conversions.ex

```diff
@@ -102,6 +103,16 @@ defmodule ElixirLS.LanguageServer.Experimental.SourceFile.Conversions do
     {:ok, range}
   end
 
+  def to_lsp(%ElixirSense.Location{} = location, %SourceFile{} = source_file) do
+    position = SourceFile.Position.new(location.line, location.column - 1)
```

Comment:
Is line 1-based now?

Reply:
If you mean `elixir_ls`, I think yes. 

there is a comment here: https://github.com/elixir-lsp/elixir-ls/blob/a3ed3666bb6e70a356eaedab6c4d5a7433deabbf/apps/language_server/lib/language_server/experimental/provider/handlers/find_references.ex#L18